app/views/kaminari/admin/_next_page.html.erb

```diff
@@ -0,0 +1,4 @@
+<li class="flex items-center">
+  <%= link_to_unless current_page.last?, crayons_icon_tag("chevron-right", aria_hidden: true, class: ("admin-pagination--inactive mx-1" if current_page.last?).to_s), url, rel: "next", remote: remote, class: "admin-pagination--link  c-link c-link--icon-alone",
```

Comment:
The formatting in these files looks odd to me, but the linter did it automagically ✨  so I choose to trust it 😄